crates/types/src/entities/coin.rs

```diff
@@ -30,8 +30,10 @@ pub struct Coin {
     pub maturity: BlockHeight,
     /// Whether a coin has been spent or not
     pub status: CoinStatus,
-    /// Which block this coin was created in
+    /// The height of the block this coin was created in
```

Comment:
Why we don't want to use `TxPointer` here? Because it will require modification of a lot of code?

Reply:
switched to tx_pointer